README.adoc

```diff
@@ -160,6 +160,15 @@ Assuming you've installed `libmagic` properly and you have
 https://golang.org/doc/install[setup a Go environment], Use the
 following command to install Pistol to `$GOPATH/bin/pistol`:
 
+==== Go 1.17 or later
+
+[source,sh]
+----
+go install github.com/doronbehar/pistol/cmd/pistol@latest
```

Comment:
Did you verify `CGO_ENABLED` is not needed with this command?

Reply:
Of course. My verification process was to just run the command on my Mac without CGO_ENABLED and then test the resulting binary.